### DISCUSSION TITLE
ERC-20 Icons

### DISCUSSION THREAD

**Discussion by Staffanox** [LedgerHQ/ledger-live-desktop#3579] [Ideas]
[x] I have checked this feature was not yet requested.

Instead of having the Ethereum icon with an accompanied letter representing an ERC-20 token it'd be great to have a standalone icon for a token, similar to the ones created for coins like Bitcoin and  Ethereum
Obviously matching all current and potential future ERC-20 tokens with a unique icon would be too much, however adding familiar tokens like : Tether,Chainlink or BAT would be a great start

**Comment by gre** [comment]
we're considering it but at the moment it's a bit challenging to provide, specifically there is no real "standard" that allow us to solve the icon of a token and that looks relatively "ok" (like we need a monochrome svg).

happy to consider if you have a suggestion

**Reply by KarmCraft** [reply]
Have you looked into http://cryptoicons.co ?

**Reply by cthepot** [reply]
@KarmCraft Thanks for the link, shared to design team.
